packages/common/src/ui/components/menus/ColorMenu/ColorMenu.tsx

```diff
@@ -62,7 +62,8 @@ export const ColorMenu: FC<ColorMenuProps> = ({
           role="button"
           aria-label={name}
           key={hex}
-          onClick={() => {
+          onClick={e => {
+            e.stopPropagation();
```

Comment:
this stops the row event from triggers. Unsure if the best place

Reply:
I think this is the right place